containers/docker/pwpush/docker-compose-ephemeral.yml

```diff
@@ -0,0 +1,47 @@
+version: '2.1'
+services:
+
+  passwordpusher:
+    image: docker.io/pglombardo/pwpush:release
+    environment:
+      PWP_DB: 'sqlite'
```

Comment:
Is this env var needed any more?

Maybe we should call out DATABASE_URL here to be completely evident to users.

Reply:
I'll test these docker-compose files after lunch today.